### PR TITLE
Use eventio v0.17.1

### PR DESCRIPTION
**Pull request by dneise**
* Can read more kinds of simtel files (e.g. where multiple runs got merged)
* Can read calibration events

### CODE REVIEW

**Comment by MaxNoe** [comment]
This needs the not yet released version 0.17.0 of pyeventio and 0.2.17 of ctapipe-extra with the calibration events test file.

**Comment by MaxNoe** [comment]
eventio 0.17.0 is now released, waiting for a new release of ctapipe-extra

**Comment by dneise** [comment]
`ctapipe/io/tests/test_simteleventsource.py::test_calibration_events` fails ...

**Comment by dneise** [comment]
Nice !

File: setup.py [hunk 1]
```diff
@@ -65,7 +65,7 @@
           'pandas',
           'bokeh>=1.0.1',
           'scikit-learn',
-          'eventio==0.11.0',
+          'eventio==0.17.0',
```
Comment (mackaiver) [inline setup.py]: should this be 0.17.1?

File: ctapipe/io/containers.py [hunk 2]
```diff
@@ -428,6 +428,7 @@ class TelescopePointingContainer(Container):
 class DataContainer(Container):
     """ Top-level container for all event information """
 
+    event_type = Field('data', "Event type")
```
Comment (mackaiver) [inline ctapipe/io/containers.py]: what are the options here? Is it data vs calibration event? Or is it observed vs simulated event?

Comment (dneise) [reply ctapipe/io/containers.py]: Ah that's indeed a good question, maybe this PR makes sense in this context.
https://github.com/cta-observatory/ctapipe/pull/978

By defining an `enum.Enum` or `enum.IntEnum` the choices are somehow obvious...

Comment (MaxNoe) [reply ctapipe/io/containers.py]: For now, data and calibration can come from eventio. But I was not really pleased with that choice. Physics and specific calibration names would be better I think.  I will change that in eventio once we agree on the names.

**Comment by MaxNoe** [comment]
Setup.py is fixed, @cta-observatory/ctapipe-core-devs please review again

**Comment by dneise** [comment]
@MaxNoe this PR was opened by me, so you could review already .. :-D

**Comment by MaxNoe** [comment]
Ah, right.

**Pull request merged**